### ISSUE TITLE
Improve RunVaspCustodian to store standard error output

### ISSUE THREAD

**Issue by mkhorton** [hackingmaterials/atomate#297]
Often XMLValidationErrors are actually the result of some other issue that's been reported in std_err.txt, it would be useful to include this in the `stored_data` also for easier inspection.

**Comment by janosh** [comment]
Came here to suggest the same thing. :)

**Comment by janosh** [comment]
Actually, can't this be done with [Fireworks trackers](https://materialsproject.github.io/fireworks/tracker_tutorial.html)? Something like

```py
from atomate.vasp.powerups import add_trackers

wf = add_trackers(wf, ["std_err.txt"], nlines=50)
```

**Comment by mkhorton** [comment]
Yes, trackers are designed for this purpose in general. I still think XMLValidationError could be improved however, because it essentially contains zero information right now.
